aspnetcore/mvc/views/tag-helpers/built-in/samples/TagHelpersBuiltIn/Pages/Products.cshtml.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.AspNetCore.Mvc.RazorPages;
+using System.Collections.Generic;
```

Comment:
I've been using the arrangement: `System` first, `Microsoft` second, app third; each group in alphabetical order.

Reply:
I'm okay with that approach. Ordering the way you suggest requires checking the **Place 'System' directives first when sorting usings** checkbox in **Tools** > **Options** > **Text Editor** > **C#** > **Advanced** > **Using Directives**. Mine was unchecked, so the **Remove and Sort Usings** option wasn't ordering this way. Let's agree to follow this convention moving forward. /cc: @Rick-Anderson

Reply:
IIRC it was enforced by an engineer on a very early PR review of mine ... back in the day ... when I was just a punk kid. 🛴 AFAIK, all of the engineering repos do it this way.

Reply:
It looks like the project templates at aspnet/templating do follow this convention. Let's do it here too.